CONTRIBUTING.md

```diff
@@ -126,6 +128,8 @@ changes haven't introduced any [Mixed Content][] warnings.
 [js]: https://github.com/MozillaFoundation/javascript-style-guide
 [synonym]: https://help.github.com/articles/closing-issues-via-commit-messages
 [PageSpeed Insights]: https://developers.google.com/speed/pagespeed/insights/
+[WebPagetest]: http://www.webpagetest.org/
+[What Does My Site Cost]: http://whatdoesmysitecost.com/
 [Tenon]: http://tenon.io/
 [ngrok]: https://ngrok.com/
 [Mixed Content]: https://developer.mozilla.org/en-US/docs/Security/MixedContent
```

Comment:
Yeah! I actually prefer this way when the .md file is also going to be viewed as plaintext, which is often the case with READMEs, etc.